src/convertToObject.js

```diff
@@ -6,7 +6,30 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cleanedString = sourceString
+    .replace(/^\s*[\r\n]/gm, '')
+    .replace(/\s*;\s*/g, ';')
+    .replace(/\s*:\s*/g, ': ')
```

Comment:
The replacement pattern `': '` in `.replace(/\s*:\s*/g, ': ')` adds a space after the colon, which might not be intended if the input string should have no spaces around colons. Consider using `':'` instead if spaces are not desired.